src/settings.js

```diff
@@ -857,6 +857,8 @@ var FETCH = 0; // If nonzero, enables emscripten_fetch API.
 
 var ASMFS = 0; // If set to 1, uses the multithreaded filesystem that is implemented within the asm.js module, using emscripten_fetch. Implies -s FETCH=1.
 
+var NO_PORTS = 0; // Disables usage of emscripten ports. Used internally when building system libraries.
```

Comment:
Yeah, I lean to the latter. We also have e.g. `NO_FILESYSTEM` for similar reasons, it makes it clear that doing the `NO_` thing is nonstandard I think.